src/components/global/ContactFormModal.jsx

```diff
@@ -0,0 +1,138 @@
+// React Imports
+import { useRef } from 'react';
+// emailjs Imports
+import emailjs from '@emailjs/browser';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+import Grid from '@mui/material/Grid';
+import TextField from '@mui/material/TextField';
+// Other Library Imports
+import { PropTypes } from 'prop-types';
+
+// TODO: Change title from "Contact Us" to "Volunteer" if using that button?
+
+const ContactFormModal = ({ showContactFormModal, setShowContactFormModal }) => {
+  const form = useRef();
+
+  const handleClose = () => {
+    setShowContactFormModal(false);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    emailjs.sendForm('service_co2agxf', 'template_20pnwni', form.current, 'AkBl59Ya3226OfPyQ').then(
+      (result) => {
+        console.log(result.text);
+      },
+      (error) => {
+        console.log(error.text);
+      }
+    );
+    e.target.reset();
+  };
+
+  return (
+    <Dialog open={showContactFormModal} onClose={handleClose}>
+      <form ref={form} onSubmit={handleSubmit} autoComplete="off">
+        <Box
+          sx={{
+            p: { xs: 0, md: 4 }
+          }}
+        >
+          <DialogTitle sx={{ display: 'flex', justifyContent: 'center' }}>Contact Us</DialogTitle>
+          <DialogContent>
+            <DialogContentText sx={{ pb: '1rem' }}>
+              For any questions or to just reach out, contact us today!
+            </DialogContentText>
+            <TextField
```

Comment:
Every `TextField` has the `autoFocus` prop, should only be the first one

<img width="882" alt="Screenshot 2023-11-19 at 8 18 53 AM" src="https://github.com/codeforpdx/codepdx_website/assets/71395931/66b5e38a-cc23-4716-9b66-631a043931d9">

But I tested with only one and it didn't seem to autofocus for me so 🤷🏻‍♂️

Reply:
I changed it. We can keep testing